### PR TITLE
Guide on RxNorm and NDC codes

### PR DESCRIPTION
**Pull request by rahul1**
This is pre-work for #1400 and #1398 . Medication are a nuanced topic in and of themselves, and are referenced from different parts of the medication workflow, so it makes sense to split into a separate doc

### CODE REVIEW

File: packages/docs/docs/medications/medication-codes.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+Unlike RXNorm, NDC operates as a flat code system and only refers to the "leaf nodes" of the RXNorm hierarchy. If a single manufacturer issues the same medication in packages of different sizes (like 25 tablets, 50 tablets, etc.), each size is assigned a unique NDC.
+
+However, there are connections between the two code systems, as a drug product can have both NDC and RXNorm codes. Below is an example of the a [CodeableConcept](/docs/fhir-basics#codeable-concepts-standarding-data) for Tylenol, with both NDC and RxNorm codes
+
+```ts
+{
+  text: 'Tylenol 325 MG Oral Tablet'
+  coding: [
+    //
```
Comment (codyebberson) [inline packages/docs/docs/medications/medication-codes.md:68]: Minor nit: add comment or remove this line

**Comment by codyebberson** [comment]
We may want to add something about FDB, as it is the more commonly used database in the wild.

--------

Me: compare and contrast the following drug databases:  rxnorm, ndb, and fdb.  in particular, i'm curious about popularity and common use cases.

ChatGPT:

1. RxNorm:

RxNorm is a standardized nomenclature for clinical drugs developed by the U.S. National Library of Medicine. It provides normalized names for clinical drugs and connects its names to many of the drug vocabularies commonly used in pharmacy management and drug interaction software, such as those in commercial databases.

RxNorm is primarily used in electronic health record (EHR) systems, enabling them to communicate and exchange information about medications effectively. It's often used in prescribing, medication administration, decision support, and other clinical software systems.

Popularity: RxNorm is widely adopted in the US healthcare system due to its integration with the U.S. National Library of Medicine and its ability to interact with other major drug databases.

2. NDB (National Drug Data File):

The NDB, also known as the National Drug File - Reference Terminology (NDF-RT), is maintained by the U.S. Department of Veterans Affairs. The NDF-RT provides information about drug characteristics, including ingredients, chemical structure, dose form, physiologic effect, mechanism of action, pharmacokinetics, and related diseases.

NDF-RT is typically used in decision support systems for medication therapy management, drug formulary checks, medication safety checks, and population-based analyses of drug effects.

Popularity: NDB/NDF-RT is less popular compared to RxNorm and FDB. However, it is still widely used within the U.S. Department of Veterans Affairs and by some EHR vendors and researchers due to its comprehensive data related to drug characteristics.

3. FDB (First Databank):

First Databank is a private company that provides drug databases covering various aspects of drug information, including drug interactions, contraindications, clinical decision support, patient education, and more. FDB's content includes information on prescription and over-the-counter medications, nutritional products, medical devices, and more.

FDB's drug information is commonly used in healthcare software applications, including CPOE systems, pharmacy systems, EHRs, and other healthcare IT systems. It's often used for decision support, electronic prescribing, medication therapy management, pharmacy automation, patient education, and more.

Popularity: FDB is very popular in both hospital and retail pharmacy settings due to its extensive data, which support a variety of clinical and operational use cases. Its commercial nature means that it has extensive coverage, including many over-the-counter medications and nutritional products not included in other databases.

In summary, the choice of a drug database often depends on the specific use case, budget, and the extent of data required. RxNorm and FDB are widely popular, with RxNorm often being preferred in clinical settings due to its standardization and free availability, while FDB is favored for its extensive, detailed content. NDB/NDF-RT has specific advantages in areas like drug effect analyses and is heavily used within the U.S. Department of Veterans Affairs.

File: packages/docs/docs/medications/medication-codes.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+This guide is dedicated to discussing two of the most significant, non-proprietary code systems in U.S. healthcare: [RXNorm](https://www.nlm.nih.gov/research/umls/rxnorm/overview.html) and [NDC](https://www.fda.gov/drugs/drug-approvals-and-databases/national-drug-code-directory).
+
+- [RXNorm](https://www.nlm.nih.gov/research/umls/rxnorm/overview.html) is an international standard endorsed by standards-compliant bodies such as the US Core, NIH, and CDC.
+- [NDC](https://www.fda.gov/drugs/drug-approvals-and-databases/national-drug-code-directory) (National Drug Code) is a U.S. specific system administered by the Food and Drug Administration (FDA)
+
+While there are other proprietary drug databases with their proprietary code systems, they will not be explored in this guide.
+
+## RxNorm
+
+RxNorm is the preferred code system of the [U.S. Core](/docs/fhir-datastore/understanding-uscdi-dataclasses) and the National Institutes of Health (NIH), as well as most FHIR profiles.
```
Comment (codyebberson) [inline packages/docs/docs/medications/medication-codes.md:26]: "preferred system of the U.S. Core" - reads weird to me

Maybe "preferred system of the US Core Medication Profile" ?

Also, reading the US Core docs, it looks like they do not use periods in "US" vs "U.S."

**Review by codyebberson — COMMENTED** [review]
A couple minor nits

We should add something about FDB, but we can do that separately.

File: packages/docs/docs/medications/medication-codes.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
+---
+keywords:
+  - rxnorm
+  - ndc
+  - drugs
+  - drug codes
```
Comment (reshmakh) [inline packages/docs/docs/medications/medication-codes.md:6]: add "medications"

File: packages/docs/docs/medications/medication-codes.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+- [RXNorm](https://www.nlm.nih.gov/research/umls/rxnorm/overview.html) is an international standard endorsed by standards-compliant bodies such as the US Core, NIH, and CDC.
+- [NDC](https://www.fda.gov/drugs/drug-approvals-and-databases/national-drug-code-directory) (National Drug Code) is a U.S. specific system administered by the Food and Drug Administration (FDA)
+
+While there are other proprietary drug databases with their proprietary code systems, they will not be explored in this guide.
+
+## RxNorm
+
+RxNorm is the preferred code system of the [US Core Medication Profile](https://hl7.org/fhir/us/core/stu3.1.1/StructureDefinition-us-core-medication.html) and the National Institutes of Health (NIH), as well as most FHIR profiles.
+
+RXNorm is a hierarchical code system, that allows for different levels of specificity, depending on the application. The RXNorm vocabulary establishes standard names and identifiers for combinations of ingredients, strengths, and dose forms. This is the information doctors typically include when they write a prescription because they often can’t know the specific product that will be used to fill it.
```
Comment (reshmakh) [inline packages/docs/docs/medications/medication-codes.md:28]: describing how these handle brand names or colloquial names might be worth a mention here.  I notice you use "Prozac" as an example.  

Also note how synonyms are handled in this code system.

**Review by codyebberson — APPROVED** [review]
